src/Entities/AvsResponse.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Rebilly\Entities;
+
+use Rebilly\Rest\Entity;
+
+/**
+ * Class AvsResponse.
+ */
+final class AvsResponse extends Entity
```

Comment:
It's not an entity, this object does not have id. You should use Resource in such classes.